### ISSUE TITLE
2025.10.15 - #51 - OKVIS2-X, Open-YOLO 3D, CoT-VLA, π0.5, RND1, SuperDec

### ISSUE THREAD

**Issue by changh95** [changh95/WeeklySpatialAI#53]
# Interesting papers

## OKVIS2-X: Open Keyframe-based Visual-Inertial SLAM Configurable with Dense Depth or LiDAR, and GNSS

- https://youtu.be/K8oZvbI7I58?si=sT1NK6rYN982dSFW
- https://github.com/ethz-mrl/OKVIS2-X
- https://arxiv.org/pdf/2510.04612
- visual, inertial, measured or learned depth, LiDAR, GNSS를 마음대로 활용할 수 있는 SLAM 시스템
- Dense volumetric map을 이용한다는게 특징. 그럼에도 submap 구조를 잘 활용해서, 9km 까지 scale이 가능함.
- Camera extrinsic online calibration도 가능

<img width="1900" height="1073" alt="Image" src="https://github.com/user-attachments/assets/63f5933f-643b-46ec-9264-76a24c8d2ea9" />

<img width="1439" height="878" alt="Image" src="https://github.com/user-attachments/assets/f1076acd-b8c3-42b7-8f54-2231d92f7bb0" />

## Open-YOLO 3D: Towards Fast and Accurate Open-Vocabulary 3D Instance Segmentation

- https://github.com/aminebdj/OpenYOLO3D
- https://arxiv.org/pdf/2406.02548
- SAM 이나 CLIP을 매 프레임마다 추출해서 multi-view reconstruction을 통해 3D instance segmentation을 하는 모델들이 많았음. 굉장히 무거운 연산들이라 속도가 많이 느림.
- 2D object detection + 3D network 두개만으로 기존의 Sota보다 16배나 빠른 속도를 얻어낼 수 있음.

<img width="2353" height="667" alt="Image" src="https://github.com/user-attachments/assets/9aa9765f-bec8-4a26-9eb7-5f42ff7f8049" />

<img width="1439" height="646" alt="Image" src="https://github.com/user-attachments/assets/9d14dec8-f819-4853-afbd-194715ab0081" />

<img width="1439" height="646" alt="Image" src="https://github.com/user-attachments/assets/eb26ec11-a68e-467d-83ff-c33f3daae793" />

**Comment by chae2chae** [comment]
### [CoT-VLA: Visual Chain-of-Thought Reasoning for  Vision-Language-Action Models](https://arxiv.org/abs/2503.22020)

<img width="334" height="549" alt="Image" src="https://github.com/user-attachments/assets/27b3c57e-7b25-44d2-b14b-5129400a84fb" />

- Problem Formulation 
  - 기존 VLA(Vision-Language-Action) model은 observation을 입력받아 바로 행동을 예측하는 구조를 사용함. 이러한 방식은 복잡한 조작 혹은 multi-step manipulation 에서 중간 추론 단계가 없어 효율적인 계획 수립이 어려움. + 대규모 unlabeled video 데이터를 사용하지 못해 시각적인 일반화 능력이 제한적임
  	- CoT-VLA 는 이 문제를 해결하기 위해 시각적 추론 단계를 명시적으로 도입
- Proposed Method 
  - CoT-VLA는 행동을 예측하기 전에 미래 subgoal image를 생성해서 이를 기반으로 행동을 결정하여 시각적 추론이 모델 내부에서 가능하도록 함. 
- Training Method & Dataset 
  - 모델은 Open-X-Embodiment(OXE) 로봇 데이터와 EPIC-KITCHENS, Something-Something V2 등의 unlabled video 데이터를 함께 사용해 사전학습(pretraining)됨. 백본(backbone)은 7B 규모의 VILA-U 모델이며, subgoal 이미지 예측 손실(L2 또는 perceptual loss)과 행동 예측 손실(cross-entropy 또는 regression loss)을 동시에 최적화하는 joint training 방식을 사용. 모델은 “다음 상태가 어떻게 생길 것인가”를 먼저 학습하고, 이어 “그 상태를 만들기 위한 행동”을 예측하도록 훈련됨.

<img width="679" height="306" alt="Image" src="https://github.com/user-attachments/assets/96dabcb0-f3fd-4462-9c0d-ab77144554bf" />



### [π0.5: a Vision-Language-Action Model with Open-World Generalization](https://arxiv.org/pdf/2504.16054)

<img width="703" height="444" alt="Image" src="https://github.com/user-attachments/assets/5022814b-e56c-446f-a869-17973e56be71" />

- Problem Formulation 
  - 실제 로봇이 학습에 사용되지 않은 가정 환경에서 길고 복잡한 조작 작업을 수행하도록 하는 것이 핵심 과제임. 기존 Vision-Language-Action(VLA) 모델은 제한된 데이터 분포에 과적합되는 경향이 있으며, 새로운 물체, 공간 배치, 언어 명령에 대한 일반화 능력이 부족함. 이러한 한계로 인해 로봇이 일상적인 환경에서 장기적이고 다단계의 조작을 수행하기 어려움.
- Proposed Method 
  -  π 0.5는 이미지, 언어 명령, 객체 위치, 그리고 고수준 subtask 정보를 하나의 통합된 시퀀스로 인코딩함. 모델은 입력으로부터 수행해야 할 subtask를 먼저 예측하고, 그 결과를 바탕으로 저수준 행동을 생성함. 저수준 행동은 flow matching 기반의 action expert 모듈을 통해 연속 제어 신호로 변환됨. 이를 통해 고수준 reasoning과 저수준 control을 분리하여 학습함으로써 복잡한 작업을 효율적으로 수행할 수 있도록 함.

- Training Method & Dataset  
	- 280k 스텝 동안 모든 데이터를 FAST tokenizer를 이용해 이산화(discretization)하여 학습함. 이후 80k 스텝 동안 flow matching 기반의 연속형 action expert로 조정함.  이 과정에서 모바일 및 비모바일 로봇 데이터, 웹 기반 시각·언어 데이터, 그리고 고수준 subtask 라벨을 함께 사용함. 이러한 multi-source 결합 학습을 통해 모델은 시각적 맥락, 언어적 명령, 동작 패턴을 동시에 이해하고 표현할 수 있게 됨.

<img width="694" height="378" alt="Image" src="https://github.com/user-attachments/assets/d4dc4bf4-6af5-4e14-bdf3-cacca869c5a0" />
